templates/docker-compose/.env.dist

```diff
@@ -4,6 +4,7 @@ COMPOSE_PROJECT_NAME=supportpal
 HTTP_REMOTE_PORT=80
 HTTPS_REMOTE_PORT=443
 MYSQL_REMOTE_PORT=3306
+SOCKETS_REMOTE_PORT=6601
```

Comment:
What's this for? Doesn't appear to be used

Reply:
was previously used with old nginx config, should remove